src/components/stepforms/widgets/SortColumn.vue

```diff
@@ -55,17 +53,37 @@ export default class SortColumnWidget extends Vue {
   errors!: ErrorObject[];
 
   @VQBModule.Getter columnNames!: string[];
-
   @VQBModule.Mutation setSelectedColumns!: MutationCallbacks['setSelectedColumns'];
 
-  sort: SortColumnType = { ...this.value };
+  created() {
+    this.update(this.value);
```

Comment:
I have the exact same concern than https://github.com/ToucanToco/weaverbird/pull/472

Reply:
not ok with that because in that case will always have a value to emit on created, the column proprety that was selected

Reply:
@davinov What do you think ?

Reply:
otherwise I could merge that PR

Reply:
Ok I see what you mean. I think the logic should be this one:
- if we do have a selected in column in the data table, then I expect the component to emit a new value with the selected column filled
- if we do NOT have a selected column in the data table, then:
  - if the value of the component is the default one, I expect the component to emit it
  - if the value of the component is not the default one, I expect the component NOT to emit anything, to avoid overriding this value

Reply:
After discussion the sortColumn widget has no need to emit this event at created to his parent because sortStepForm has already this data